### DISCUSSION TITLE
Solid GPT: A personalized Solid assistant 🤖

### DISCUSSION THREAD

**Discussion by danielbakas** [solid-contrib/getting-started#25] [Show and tell]
Hi everyone!

Just wanted to share with everybody this **Solid GPT** we made at Semantyk 🤓

**HERE'S THE LINK SO YOU CAN TRY IT OUT: https://chat.openai.com/g/g-3HPT4dynB-solid**

**"Solid"** (named after the project) is a multilingual GPT expert in the Solid Project and Semantic Web technologies and standards 📄

It provides comprehensive information on technical and strategic aspects of Solid, and is designes as a resource for websites, specifications, and community forum topics 🔍

It adapts its responses to most available languages and communications styles based on the user's preference 🌎

The GPT maintains a balanced tone, being neutral or slightly formal yet approachable. It focuses on providing accurate, valuable information, avoiding speculative content 🪷

It adjusts its communication to the user's knowledge level and language, offering technical explanations, strategic overviews, and guiding beginners through the learning process of getting to know Solid (from the specs, to the frameworks and more) 🏔️

It shares real-world applications and success stories to provide practical insights. 🎖️

"Solid" guides users to appropriate resources or experts for queries beyond its scope, clearly indicating the limits of its knowledge, especially on recent developments. 💡

"Solid" includes capabilities like writing TURTLE, SPARQL, OWL, RDF and many more semantic web technologies, interactive problem-solving, and general inquiry assistance. 🧑🏽‍💻

It aims to be an engaging and informative resource for anyone interested in Solid technology, regardless of their preferred language or previous experience. It can adapt dynamically to the level of complexity the user is interested in for a given conversation. 🧮

We would love to open up an invitation for anyone to try it out and take it for a spin 🚀 

We would also love to invite you to share your feedback, how we can improve it, and what you would like for us to add or fix to this GPT so we can make it even better 🥳

Thank you and cheers! 🎉

**Disclaimer:** It's important to know that you'll be able to access "Solid" GPT with your ChatGPT Plus (GPT-4) subscription until OpenAI decides to open up GPTs to the public (hopefully soon 🤞🏽)

**Comment by KennethLeeJE8** [comment]
hey, would I still be able to access the GPT without a GPT-4 subscription? I would like to try it out

**Reply by danielbakas** [reply]
Hi @KennethLeeJE8! I think GPTs are currently available only for GPT Plus subscribers 😭

Feel free to double check and let me know 😄
